scripts/jenkins/ardana/ansible/install-crowbar.yml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Again something similar why is a crowbar file in a ardana related folder?

Reply:
As I was saying in the previous comment, the Crowbar ECP CI is based heavily on the Ardana baseline (ansible playbooks). We need to do this refactoring work, but it's not on our list of priorities (see https://jira.suse.de/browse/SCRD-8475)